cmd/eksctl-anywhere/cmd/importimages.go

```diff
@@ -0,0 +1,197 @@
+		return fmt.Errorf("error importig image %s: %v", bundle.CertManager.Controller.URI, err)
+	}
+	if err := importImage(context, de, bundle.CertManager.Webhook.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.CertManager.Webhook.URI, err)
+	}
+
+	if err := importImage(context, de, bundle.Cilium.Cilium.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Cilium.Cilium.URI, err)
+	}
+	if err := importImage(context, de, bundle.Cilium.Operator.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Cilium.Operator.URI, err)
+	}
+
+	if err := importImage(context, de, bundle.ClusterAPI.Controller.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.ClusterAPI.Controller.URI, err)
+	}
+	if err := importImage(context, de, bundle.ClusterAPI.KubeProxy.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.ClusterAPI.KubeProxy.URI, err)
+	}
+
+	if err := importImage(context, de, bundle.ControlPlane.Controller.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.ControlPlane.Controller.URI, err)
+	}
+	if err := importImage(context, de, bundle.ControlPlane.KubeProxy.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.ControlPlane.KubeProxy.URI, err)
+	}
+
+	if err := importImage(context, de, bundle.Docker.KubeProxy.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Docker.KubeProxy.URI, err)
+	}
+	if err := importImage(context, de, bundle.Docker.Manager.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Docker.Manager.URI, err)
+	}
+
+	if err := importImage(context, de, bundle.EksD.KindNode.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.EksD.KindNode.URI, err)
+	}
+
+	if err := importImage(context, de, bundle.Eksa.CliTools.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Eksa.CliTools.URI, err)
+	}
+
+	if err := importImage(context, de, bundle.Flux.HelmController.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Flux.HelmController.URI, err)
+	}
+	if err := importImage(context, de, bundle.Flux.KustomizeController.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Flux.KustomizeController.URI, err)
+	}
+	if err := importImage(context, de, bundle.Flux.NotificationController.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Flux.NotificationController.URI, err)
+	}
+	if err := importImage(context, de, bundle.Flux.SourceController.URI, endpoint); err != nil {
+		return fmt.Errorf("error importig image %s: %v", bundle.Flux.SourceController.URI, err)
+	}
+
+	if err := importImage(context, de, bundle.VSphere.ClusterAPIController.URI, endpoint); err != nil {
```

Comment:
Small thing, but shouldn't these provider specific?  vsphere is our only prod provider right now, but just curious...

Reply:
I thought this command would be used once and would push all images from the version bundle to a local registry.
In practice we just need the shared and vsphere specific, right? I've removed the aws images